src/components/JoinButton.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components/macro";
+
+const JoinButton = ({ isJoined, toggleJoin }) => {
+  const handleJoin = () => {
+    toggleJoin();
+  };
+
+  return (
+    <StyledJoinButton isJoined={isJoined} onClick={handleJoin}>
```

Comment:
You could export the styled component and let this button be dump. onClick is a standard prop of a styled component button, so you don't have to declare it, just call the button with onClick property, same with the child. 
If you have any questions about it, you could write me on slack.

Reply:
Thx for the tip! Think I got your point, but wanna keep my button component – even if it's small... 
However I streamlined the code a bit:
`const JoinButton = ({ isJoined, toggleJoin }) => {
  return (
    <StyledJoinButton isJoined={isJoined} onClick={toggleJoin}>
      {isJoined ? "WITHDRAW" : "JOIN ACTIVITY"}
    </StyledJoinButton>
  );
};`